decidim-proposals/app/events/decidim/proposals/admin/update_proposal_category_event.rb

```diff
@@ -0,0 +1,48 @@
+# frozen-string_literal: true
+
+module Decidim
+  module Proposals
+    module Admin
+      class UpdateProposalCategoryEvent < Decidim::Events::BaseEvent
```

Comment:
I recommend using `SimpleEvent`

Reply:
when it was implemented `SimpleEvent` and `AuthorEvent` where not in `master` i'll refactor it